Xamarin.PropertyEditing.Mac/Controls/Custom/NumericTextField.cs

```diff
@@ -93,9 +93,8 @@ public static bool CheckIfNumber (string finalString, ValidationType mode, bool
 
 		public static bool ValidateDecimal (string finalString, bool allowNegativeValues)
 		{
-			double value;
 			//Checks parsing to number
-			if (!double.TryParse (finalString, System.Globalization.NumberStyles.Any, System.Globalization.CultureInfo.CurrentUICulture, out value))
+			if (!double.TryParse (finalString, System.Globalization.NumberStyles.Any, System.Globalization.CultureInfo.CurrentUICulture, out var value))
```

Comment:
Going forward, let's not make changes like this unless we're actually touching this line, we're just making the real history of this line harder to find and making it harder to review for nearly no benefit.